source/manual/alerts/email-alert-api-unprocessed-content-changes.html.md

```diff
@@ -63,6 +63,10 @@ ContentChange.where("created_at < ?", 10.minutes.ago).where(processed_at: nil).m
 
 - [content_change_failed_emails][]
 
+### Requeue content changes and messages over 1-hour old for which jobs have been lost
```

Comment:
Could we duplicate this rake task and text in the [unprocessed messages doc](https://docs.publishing.service.gov.uk/manual/alerts/email-alert-api-unprocessed-messages.html)?